README.md

```diff
@@ -5,39 +5,38 @@ Welcome to the Bluetooth Numbers Database, an online repository containing metad
 
 **Bluetooth Numbers Database** is maintained by Nordic Semiconductor ASA.
 
-## Scope
+## Background
 
-The Bluetooth Specification makes heave use of a Universally Unique Identifier field, known **UUID** for short, to define multiple key elements in the spec, such as a device's manufacturer information. In the development of our own tools, it was discovered many of these definitions were duplicated and redefined in a variety of projects, hence the need to unify that body of work into a single, cohesive project representing a [Single Source of Truth](https://en.wikipedia.org/wiki/Single_source_of_truth).
+During the development of various Bluetooth tools at Nordic, many of the assigned numbers from the Bluetooth specificaiton have been taken into use in different areas, which has lead to duplication of effort between the different tools. Hence it has been seen as beneficial to unify the work efforts by using a single, cohesive project representing a [Single Source of Truth](https://en.wikipedia.org/wiki/Single_source_of_truth).
 
-In addition, it is common-case within the Bluetooth community, and covered by the specification, to extend the SIG-defined elements with proprietary definitions that do not overlap with the standard. This is perfectly allowed, but without a single place to check against whether your proposed UUID is used by others, we run the risk of having multiple uses for the same identifier.
+In addition to looking up officially defined numbers, it is common to extend the SIG-defined numbers with proprietary definitions that do not overlap with the standard. Such extension is permitted, but unless there is a place to check against whether your proposed UUID is used by others, we run the risk of having multiple uses for the same identifier.
```

Comment:
I like this a lot better than my own writing.

Reply:
👍